### DISCUSSION TITLE
Libev on python - pyev

### DISCUSSION THREAD

**Discussion by jason79461385** [1101-GS4719-C/Python-Community-GS4719-C#508] [General]
在C語言上，可以透過libev來達成限制使用者在指定時間內完成某項指令，例如：可以限制使用者必須在5.5秒內input所需要的東西。
![image](https://user-images.githubusercontent.com/75102829/159637672-42fe697a-dff5-4b23-9f1c-10536883b37d.jpeg)

而在python上也可透過pyev這個套件來完成libev上的功能，使得開發者在python上能夠有更具互動性質的功能。

**Comment by yucheng1028** [comment]
雖然看不懂，但感覺是個電神

**Comment by ghost** [comment]
雖然看不懂，但感覺是個大佬(#

**Comment by hapiwangy** [comment]
這類型的技術似乎是應用在限時認證碼之類的地方，但也很好奇是否有更多的其他用途。

**Reply by jason79461385** [reply]
之前做過的東西是可以用在下棋上，來限制下棋者下這步棋要在多久內下完

**Comment by HuanYuuuuuu** [comment]
感覺這項技術可以拿來設計成遊戲之類的

**Reply by jason79461385** [reply]
之前有用過西洋棋的遊戲，我的個人專案那好像有公開的樣子

**Comment by PoooYen** [comment]
感覺快問快答的設計就需要這種功能!!
